db/models/Cart.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db");
+const { Sequelize } = db;
+
+const Cart = db.define("cart", {
+  id: {
+    type: Sequelize.UUID,
+    defaultValue: Sequelize.UUIDV4,
+    primaryKey: true
+  },
+  status: {
+    type: Sequelize.STRING,
```

Comment:
Should we try to use Sequelize.ENUM(Sequelize.STRING)?

Reply:
I've been advised by a Sam, and whoever graded my senior checkpoint not to use Enum. ENUM fields do not get dropped when you sync the db, even with the force flag on. That would make any changes to that field problematic to deal with.